app/src/main/java/fr/neamar/kiss/result/SearchResult.java

```diff
@@ -48,6 +48,7 @@ public View display(Context context, int position, View v) {
     public void doLaunch(Context context, View v) {
         boolean exceptionThrown = false;
         Intent search = new Intent(Intent.ACTION_WEB_SEARCH);
+		search.setSourceBounds(v.getClipBounds());
```

Comment:
Turns out it currently always crashes when you use the "search"-button on the keyboard, guess who never uses that…